src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.ts

```diff
@@ -22,6 +22,8 @@ export class UsageBarComponent implements OnChanges {
   decimals = 0;
   @Input()
   calculatePerc = true;
+  @Input()
+  title = 'usage';
```

Comment:
For example, it should be translated this:
```suggestion
  title = $localize`usage`;
```